esp/src/eclwatch/LZBrowseWidget.js

```diff
@@ -202,15 +202,25 @@ define([
             }
 
             if (!this.dropZoneMachineSelect.initalized) {
+                var selectedOS;
                 this.dropZoneMachineSelect.init({
                     DropZoneMachines: true,
                     callback: function (value, row) {
+                        var path = targetRow.machine.Directory.indexOf("/");
                         targetRow.machine.Name = value
                         targetRow.machine.Netaddress = value
                         if (context.dropZoneFolderSelect) {
                             context.dropZoneFolderSelect._dropZoneTarget = targetRow;
                             context.dropZoneFolderSelect.defaultValue = "/"
-                            context.dropZoneFolderSelect.loadDropZoneFolders();
+                            if (path > -1) {
```

Comment:
line #209 is looking for the presence of a Linux variant / if it detects it will be a positive number which in this case the first if block #216 will execute. Line # 214 however is duplicate code which is not needed will adjust.